### PR TITLE
Tox & Jenkinsfile

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,18 +2,25 @@ BASE_IMAGE := registry.fedoraproject.org/fedora:29
...
 PY_PACKAGE := ogr
 OGR_IMAGE := ogr
+OGR_TESTS_IMAGE := ogr-tests
 
 build: recipe.yaml
-	sudo ansible-bender build --build-volumes $(CURDIR):/src:Z -- ./recipe.yaml $(BASE_IMAGE) $(OGR_IMAGE)
+	ansible-bender build --build-volumes $(CURDIR):/src:Z -- ./recipe.yaml $(BASE_IMAGE) $(OGR_IMAGE)
+
+build-tests: recipe-tests.yaml
+	ansible-bender build -- ./recipe-tests.yaml $(OGR_IMAGE) $(OGR_TESTS_IMAGE)
```
Comment (TomasTomecek) [inline Makefile]: <3 glad you didn't give up on it

Comment (jpopelka) [reply Makefile]: it's just old commit :-)

**Comment by jpopelka** [comment]
ready for review

**Comment by jpopelka** [comment]
> Leaving the final review for our QE guru.

merging so we have the Jenkinsfile in master, @rpitonak @jscotka PTAL anyway

**Comment by jpopelka** [comment]
[CI is green too](https://ci.centos.org/job/ogr-pr/3/), merging so we have the Jenkinsfile in master, @rpitonak @jscotka PTAL anyway

**Pull request merged**